xla/debug_options_flags.cc

```diff
@@ -203,6 +203,7 @@ DebugOptions DefaultDebugOptionsIgnoringFlags() {
   opts.set_xla_gpu_ensure_minor_dot_contraction_dims(false);
   opts.set_xla_gpu_filter_kernels_spilling_registers_on_autotuning(true);
   opts.set_xla_gpu_llvm_verification_level(0);
+  opts.set_xla_gpu_enable_cudnn_layer_norm(false);
```

Comment:
Put this flag under `xla_gpu_enable_cudnn_fmha` and `xla_gpu_fused_attention_use_cudnn_rng`, since cudnn layer norm is a similar feature. And same in xla.proto.

Reply:
> Put this flag under `xla_gpu_enable_cudnn_fmha` and `xla_gpu_fused_attention_use_cudnn_rng`, since cudnn layer norm is a similar feature. And same in xla.proto.

You didn't move the flag in debug_options_flag.cc yet.